<launch>
  <!-- Camera -->
  <node pkg="bluefox2" type="bluefox2" name="camera" output="screen" >
    <param name="fps" type="double" value="3.0" />
    <param name="gain" type="double" value="0.0" />
    <param name="exposure_time_us" type="int" value="10000" />
    <param name="use_auto_exposure" type="bool" value="false" />
    <param name="use_color" type="bool" value="false" />
    <param name="use_hdr" type="bool" value="false" />
    <param name="has_hdr" type="bool" value="false" />
    <param name="use_inverted" type="bool" value="false" />
    <param name="use_binning" type="bool" value="false" />
    <param name="use_split_image" type="bool" value="false" />
    <param name="use_stereo" type="bool" value="true" />
    <param name="serial_left"  type="string" value="25001110" />
    <param name="serial_right"  type="string" value="25000855" />
  </node>
</launch>
